.github/workflows/tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  py27:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 2.7
```

Comment:
I don't think we need to keep py27 env for testing. I believe it can be dropped.
Maybe you could add job for py39,310 and 311

Reply:
Thanks! Changes to the above versions.